operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/DefaultEventHandler.java

```diff
@@ -146,7 +124,13 @@ private void executeBufferedEvents(String customResourceUid) {
               latestCustomResource.get(),
               retryInfo(customResourceUid));
       log.debug("Executing events for custom resource. Scope: {}", executionScope);
-      executor.execute(new ExecutionConsumer(executionScope, eventDispatcher, this));
+      executor.execute(() -> {
```

Comment:
Nice!

Reply:
Actually, I'm probably going to revert that change so that we can override `toString` to get more precise debugging information on the Runnable… 🥲

Reply:
What I meant eliminating the ExecutionConsumer is elegant here

Reply:
Yes, I understood that but like I said, I actually need to re-introduce some equivalent form of it :)